### DISCUSSION TITLE
How to integrate for Trilium?

### DISCUSSION THREAD

**Discussion by phanhavn** [waterovo/trilium-simple-mind-map#8] [Q&A] [answered]
Hi there,
Thank you so much for your genius feature. This feature I waiting so long.
Could you help me know how to integrate this feature for Trillium.
Thank you so much!

**Comment by waterovo** [comment, marked as answer]
Import notes, just like the link below, and uncheck the safe import option.
https://github.com/zadam/trilium/wiki/Markdown#file-import
This passage is translated by translation software. If you don't understand anything, please continue to communicate.

**Reply by waterovo** [reply]
After importing the notes, you can create a new simple-mind-map note, just like the link below.
https://github.com/zadam/trilium/wiki/Template#instance-note

**Reply by phanhavn** [reply]
Thank you so much! I'm following your guidelines, and it's working!
